tests/manage/mcg/test_bucket_creation.py

```diff
@@ -178,3 +180,27 @@ def test_duplicate_bucket_creation(
                 logger.info(
                     f"Create duplicate bucket {bucket_name} failed as" " expected"
                 )
+
+    @bugzilla("2179271")
+    @pytest.mark.parametrize(
+        argnames="amount,interface",
+        argvalues=[
+            pytest.param(*[1, "OC"], marks=[tier1]),
+        ],
+    )
+    def test_check_for_error_in_rook_op_logs(self, bucket_factory, amount, interface):
```

Comment:
The test case name is too generic. we are checking for the bucket notification error, can we change the test name according to it?